frontend/src/Activity/Queue/Queue.js

```diff
@@ -139,10 +140,8 @@ class Queue extends Component {
       allUnselected,
       selectedState,
       isConfirmRemoveModalOpen,
-      isPendingSelected,
-      items
+      isPendingSelected
     } = this.state;
-
```

Comment:
Can you add back this line break, then good.

Reply:
Done.